app/models/user.rb

```diff
@@ -2,11 +2,11 @@
 #
 # Table name: users
 #
-#  id         :integer          not null, primary key
+#  id         :bigint(8)        not null, primary key
 #  created_at :datetime         not null
 #  updated_at :datetime         not null
 #
 
 class User < ApplicationRecord
-  has_many :lots
+  has_many :bids
```

Comment:
we want both `lots` and `bids` relation
lots relation must have `dependent` option